mimic/model/nova_objects.py

```diff
@@ -936,6 +938,37 @@ def request_action(self, http_action_request, server_id, absolutize_url):
                                          " while it is in task state other than active",
                                          http_action_request))
 
+        elif 'createImage' in action_json:
```

Comment:
For future reference, we should probably be splitting this method up into different pieces based on these "action verbs".